src/ui/connection_profiles.ui

```diff
@@ -2337,51 +2364,98 @@
-             <widget class="QLineEdit" name="character_password_entry">
+            <item row="0" column="1">
+             <widget class="QLineEdit" name="login_entry">
               <property name="toolTip">
-               <string>Characters password. Note that the password isn't encrypted in storage</string>
+               <string>The characters name</string>
               </property>
               <property name="echoMode">
-               <enum>QLineEdit::Password</enum>
+               <enum>QLineEdit::Normal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
-             <widget class="QCheckBox" name="autologin_checkBox">
-              <property name="toolTip">
-               <string>With this enabled, Mudlet will automatically start and connect on this profile when it is launched</string>
-              </property>
-              <property name="layoutDirection">
-               <enum>Qt::LeftToRight</enum>
-              </property>
-              <property name="text">
-               <string>Auto connect this profile on startup</string>
-              </property>
+             <widget class="QWidget" name="widget_options_checkBoxes" native="true">
+              <layout class="QHBoxLayout" name="horizontalLayout_6">
+               <property name="leftMargin">
+                <number>0</number>
+               </property>
+               <property name="topMargin">
+                <number>0</number>
+               </property>
+               <property name="rightMargin">
+                <number>0</number>
+               </property>
+               <property name="bottomMargin">
+                <number>0</number>
+               </property>
+               <item>
+                <widget class="QCheckBox" name="autologin_checkBox">
+                 <property name="sizePolicy">
+                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
+                   <horstretch>0</horstretch>
+                   <verstretch>0</verstretch>
+                  </sizepolicy>
+                 </property>
+                 <property name="toolTip">
+                  <string>With this enabled, Mudlet will automatically start and connect on this profile when it is launched</string>
+                 </property>
+                 <property name="layoutDirection">
+                  <enum>Qt::LeftToRight</enum>
+                 </property>
+                 <property name="text">
+                  <string>Connect automatically</string>
+                 </property>
+                </widget>
+               </item>
+               <item>
+                <widget class="QCheckBox" name="discord_optin_checkBox">
+                 <property name="enabled">
+                  <bool>false</bool>
+                 </property>
+                 <property name="toolTip">
+                  <string>Enable Discord integration (not supported by game)</string>
```

Comment:
This makes it sound as if it could not be activated. As most games have this set, it could discourage users from investigating and finding the possibilities of Lua API.
Maybe write "not immediately supported", "not directly supported", "not natively supported" or something alike.

Reply:
I think you are missing the point of this checkbox which is disabled if the game doesn't support it - it's about not misleading the user if the game does not support Discord via GMCP/MSDP.

If they tick Discord and nothing happens, they will think it's broken. That is bad user experience. That's why Mudlet will keep a list of known working games and disable the tickbox if ticking the box isn't gonna do anything. But on the plus side, we allow for workarounds via the API so those games aren't completely locked out!

Reply:
Maybe it is a misunderstanding. I don't suggest to activate that checkbox. Just add a word to the tooltip.

See some users won't read release notes. Their first experience of Discord will be to see the deactivated option there. Maybe they are intrigued and read the tooltip. I just want to prevent them thinking 'Oh this is only possible for a select few games and not mine and there is nothing that can be done about it' and they ignore it afterwards.

Reply:
It's true though, most can't do anything, but curious ones will figure it out, write a script and people will get it working that way.

Otherwise if we're vague about this people will think getting a script for this is kosher - but it's not; we want to encourage a fuss-free experience where you just tick the box and you're done, yeah?